src/libtimed/models.py

```diff
@@ -259,6 +259,7 @@ class Activities(BaseModel):
     attributes = [
         ("from-time", datetime.now(), transforms.Time),
         ("to-time", None, transforms.Time),
+        ("transferred", False, transforms.Type(bool)),
```

Comment:
```suggestion
        ("transferred", False, transforms.Type(bool, False)),
```
i have no clue what i'm reviewing tho

Reply:
```py
src/libtimed/models.py:        ("active", None, transforms.Type(bool)),
src/libtimed/models.py:        ("active", None, transforms.Type(bool)),
src/libtimed/models.py:        ("transferred", False, transforms.Type(bool)),
```

```py
class Type(BaseTransform):
    """Transform for types."""

    def __init__(
        self, type: TypingType, allow_none: bool = True, pipe: Callable = lambda x: x
    ):
        self.type = type # data type
        self.pipe = pipe # lambda function 
        self.allow_none = allow_none
```

doesn't seem like it

Reply:
well, ig we could set allow_none to false but the api technically accepts it

Reply:
oh, that makes sense :+1: